content/en/agent/configuration/network.md

```diff
@@ -412,3 +412,6 @@ If you are installing the Datadog Operator in a Kubernetes environment with limi
 [23]: /llm_observability/
 [24]: /network_monitoring/cloud_network_monitoring/
 [25]: /universal_service_monitoring/
+[26]: /metrics/
+[27]: /service_checks/
```

Comment:
This link doesn't seem to go anywhere - should it be going to https://docs.datadoghq.com/developers/service_checks/?

Reply:
Ah, yes, thanks for catching that. It should go to the link you found. Let me update.